### ISSUE TITLE
Bind error when trying to create the db while starting a new WordPress site

### ISSUE THREAD

**Issue by rpkoller** [drud/ddev#1353]
**Describe the bug**
I wanted to give ddev a try. When I tried to spin up a new WordPress site like described in the WordPress quickstart the creation of the database returns an error _Cannot create container for service db: invalid mount config for type "bind": bind source path does not exist_ 

**To Reproduce**
1. I've downloaded https://wordpress.org/wordpress-4.9.9.zip
2. Created a folder `wptest` in `~/sites`
3. Unpacked the archive within `wptest`
4. Ran `ddev init` with the suggested defaults
5. Ran `ddev start`

**Expected behavior**
I would have expected that ddev would have been able to start in the basic setup 

**Screenshots**
When i've started the first time the output looked like that: 
https://gist.github.com/rpkoller/a4b080b4d60e011e8e99e01af793cb35
Afterwards I've removed the project: 
https://gist.github.com/rpkoller/cd193380ed1d9197b0caa320a662affb
and restarted: 
https://gist.github.com/rpkoller/04fa808994422711968ee2ecdcd19283

**Version and configuration information (please complete the following information):**
 - Host computer OS and Version:  macOS High Sierra (10.13.6)
 - Docker version information (use `docker version`) and insert it here:

```Client: Docker Engine - Community
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:47:43 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.0
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       4d60db4
  Built:            Wed Nov  7 00:55:00 2018
  OS/Arch:          linux/amd64
  Experimental:     true
```

 - ddev version information (use `ddev version`)

```bgsync        	drud/ddev-bgsync:v1.5.0
cli           	v1.5.0
commit        	v1.5.0
db            	drud/ddev-dbserver:v1.5.0-10.2
dba           	drud/phpmyadmin:v1.5.0
ddev-ssh-agent	drud/ddev-ssh-agent:v1.5.0
docker        	18.09.0
docker-compose	1.23.2
domain        	ddev.local
router        	drud/ddev-router:v1.5.0
web           	drud/ddev-webserver:v1.5.0
```

 - config.yaml contents for the misbehaving project
```APIVersion: v1.5.0
name: wptest
type: wordpress
docroot: ""
php_version: "7.1"
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.2"
webcache_enabled: false
provider: default
```
 - Do you have any custom configuration (nginx, php, mysql) in the .ddev folder? If so, have you tried without them? I've used no custom configuration 

**Additional context**
In the Docker settings the path `/Users/myuser/Sites` is set in the file sharing tab. Technically the binding should be possible within that directory. :/

**Comment by rfay** [comment]
I think if you'll just share /Users (which is shared by default) or reset your docker to factory settings you'll have it working. 

It appears that you have it as "Sites" in the docker share and as "sites" in your filesystem. You and I know that they're the same on a case-insensitive macOS filesystem, but Docker doesn't know that. Could you just share /Users and see how it goes?

**Comment by rpkoller** [comment]
@rfay Yep you are right. Turns out resetting the factory settings, the file sharing settings in particular did the trick. But that way Docker is monitoring the whole Users folder, a scenario I tried to prevent before. :/ Because that caused a high cpu load and the fans spinning steadily when using docker e.g. with ddev, when i tried evaluate with a Drupal instance  a few months ago. Think I have to dig a bit deeper, into the naming thing you mentioned, and also observe how Docker is acting nowadays with the new version as well as the latest ddev version. Back then it was hardly usable on my computer. :/ But will revisit and play around a bit. Anyway `ddev start` works now as expected and I know now for the future. Thanks! Will play around with ddev running WordPress and Drupal now a bit again. Thanks again for the fast help and have relaxed holidays!
